gamestonk_terminal/cryptocurrency/extra/gasnow_model.py

```diff
@@ -0,0 +1,27 @@
+import pandas as pd
+import requests
+
+
+def get_gwei_fees() -> pd.DataFrame:
+    """Returns the most recent Ethereum gas fees in gwei
+
+    Parameters
+    ----------
+
+    Returns
+    -------
+    pd.DataFrame
+        gas prices
+    """
+
+    r = requests.get("https://www.gasnow.org/api/v3/gas/price").json()["data"]
```

Comment:
you should check for status code from request. E.g.

r = requests.get("asdad")

if r.status_code == 200:
  retrieve data
otherwise:
  flag error